pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509</version>
+    <version>1.609.1</version>
```

Comment:
> Experimental may become not experimental.

Of course, and I hope it will, but it is likely some months before we can confidently release a non-beta 1.0 of all this, at which point 1.609.x will not only be an LTS, it will be an _unsupported_ LTS. Whatever 1.565 users are using Literate today, presumably they are happy enough with the current releases, because there has not been one of Literate since Mar 2014.